apps/vpnmanager/src/lib/email/sender.ts

```diff
@@ -0,0 +1,34 @@
+import sgMail, { MailDataRequired } from "@sendgrid/mail";
+import * as Sentry from "@sentry/nextjs";
+
+import { emailKeyTemplate } from "./templates";
+
+interface MailSender {
+  recipient: string;
+  key: string;
+}
+
+export async function sendVpnKeyEmail({ recipient: to, key }: MailSender) {
+  try {
+    const sendGridApiKey = process.env.VPN_MANAGER_SENDGRID_API_KEY as string;
+    if (!sendGridApiKey) {
```

Comment:
Mostly looks good to me @koechkevin, on error handling I see in certain cases we're throwing the actual error & others returning `null`.  Isn't it better to explicitly throw the errors + message